src/SomfySynergy.js

```diff
@@ -30,6 +30,8 @@ class SomfySynergy {
       socket =>
         new Promise((resolve, reject) => {
           socket._socket.setKeepAlive(true, 5000);
```

Comment:
Importing `net-keepalive` enables the use of `socket.setKeepAlive`. In this case, the interval is being set to 5000. Why do you think we also need to set it to 1000 via `setKeepAliveInterval`?

Reply:
There are three "major" parameters when dealing with keep-alive:

1. `TCP_KEEPIDLE ` - aka. `initialDelay` - provided by nodejs
2. `TCP_KEEPINTVL` - aka. `interval` - provided by `net-keepalive`
3. `TCP_KEEPCNT` - aka. `count` - provided by `net-keepalive`

> Importing net-keepalive enables the use of socket.setKeepAlive

`socket.setKeepAlive` is provided by node and to use it you do not need `net-keepalive` to use it.
If you look at the documentation for [`socket.setKeepAlive([enable][, initialdelay])`](https://nodejs.org/api/net.html#net_socket_setkeepalive_enable_initialdelay) the second parameter is `initialDelay` and not `interval`.

From `net-keepalive` documentation: 
> Provides high-level access to socket options like TCP_KEEPINTVL and TCP_KEEPCNT

The values by default vary from system to systme as they are configurable at he system level.

Hope this clears some confusion for you :)

Reply:
Thanks for the explanation! I was indeed very confused.

I’ll merge this and release a new version when I find my way back to a computer.